Sources/Commandant/Option.swift

```diff
@@ -193,6 +193,47 @@ public func <| <T: ArgumentProtocol, ClientError>(mode: CommandMode, option: Opt
 	}
 }
 
+/// Evaluates the given option in the given mode.
+///
+/// If parsing command line arguments, and no value was specified on the command
+/// line, `nil` is used.
+public func <| <T: ArgumentProtocol, ClientError>(mode: CommandMode, option: Option<[T]?>) -> Result<[T]?, CommandantError<ClientError>> {
+	let key = option.key
+	switch mode {
+	case let .arguments(arguments):
+		var stringValue: String?
+		switch arguments.consumeValue(forKey: key) {
+		case let .success(value):
+			stringValue = value
+			
+		case let .failure(error):
+			switch error {
+			case let .usageError(description):
+				return .failure(.usageError(description: description))
+				
+			case .commandError:
+				fatalError("CommandError should be impossible when parameterized over NoError")
+			}
+		}
+	
+		guard let unwrappedStringValue = stringValue else {
+			return .success(option.defaultValue)
+		}
+		let components = unwrappedStringValue.components(separatedBy: ",").map { $0.trimmingCharacters(in: .whitespacesAndNewlines) }
```

Comment:
It's not splitting on `", "`—it's splitting on either `,` or ` `. So `mac,ios`, `mac, ios`, and `mac ios` all mean the same thing.